grammar.js

```diff
@@ -802,12 +802,18 @@ module.exports = grammar({
       '{',
       repeat(seq(
         optional('static'),
-        $.method_definition,
+        choice(
+          $.method_definition,
+          $._public_field_definition
+        ),
         optional(';')
       )),
       '}'
     ),
 
+    // TODO this should be a property_name
```

Comment:
Ah I mean this should be a larger restructuring. I believe the ES6 spec calls this a `PropertyName`.